src-ts/tools/learn/my-certificate/MyCertificate.tsx

```diff
@@ -0,0 +1,55 @@
+import { FC, useContext } from 'react'
+import { NavigateFunction, Params, useNavigate, useParams } from 'react-router-dom'
+
+import { IconOutline, profileContext, ProfileContextData } from '../../../lib'
+import { CoursesProviderData, useCoursesProvider } from '../learn-lib'
+
+import { ActionButton } from './action-button'
+import { Certificate } from './certificate'
+import styles from './MyCertificate.module.scss'
+
+interface MyCertificateProps {
+}
+
+const MyCertificate: FC<MyCertificateProps> = (props: MyCertificateProps) => {
+    const routeParams: Params<string> = useParams()
+    const { profile }: ProfileContextData = useContext(profileContext)
+    const navigate: NavigateFunction = useNavigate()
+
+    const {
+        course,
+    }: CoursesProviderData = useCoursesProvider(routeParams.certification)
+
+    function handleBackBtnClick(): void {
+        navigate(-1)
+    }
+
+    return (
+        <div className={styles['wrap']}>
+            <div className={styles['content-wrap']}>
+                <div className={styles['btns-wrap']}>
+                    <ActionButton
+                        icon={<IconOutline.ChevronLeftIcon />}
+                        onClick={handleBackBtnClick}
+                    />
+                </div>
+                <div className={styles['certificate-wrap']}>
+                    <Certificate
+                        course={course?.title}
+                        userName={[profile?.firstName, profile?.lastName].filter(Boolean).join(' ')}
+                        tcHandle={profile?.handle}
+                        provider={course?.provider}
+                        completedDate={'2022-04-27'}
```

Comment:
This static data should be replaced with the `completedDate` from the API call.

Reply:
@testflyjets api is not integrated in this branch. let's complete the other PR and I'll merge it into this one to do api integration as well

Reply:
Okay, didn't realize that.